.evergreen/run-prebuild.sh

```diff
@@ -3,7 +3,9 @@
 set -o errexit  # Exit the script with error if any of the commands fail
 set -o xtrace
 
-source "${PROJECT_DIRECTORY}/.evergreen/init-nvm.sh"
+source "${PROJECT_DIRECTORY}/.evergreen/init-node-and-npm-env.sh"
+
+export PATH="/opt/mongodbtoolchain/v2/bin:$PATH"
```

Comment:
Since we didn't know what it was for when we removed it, we should document it here
```suggestion
# mongodbtoolchain is necessary for building on Windows
export PATH="/opt/mongodbtoolchain/v2/bin:$PATH"
```
But also, should we only add this to the path on windows hosts?

Reply:
There's nothing wrong with adding additional values to the path on other hosts, I think we should leave it for simplicity.